test/evergreen.yml

```diff
@@ -5427,6 +5427,47 @@ tasks:
         vars:
           test-name: log.wtperf
 
+    #####################################
+    # Performance Tests for chunk cache #
+    #####################################
+
+  - name: perf-test-chunk-cache
+    tags: ["chunk-cache-perf"]
+    depends_on:
+      - name: compile
+    commands:
+      - func: "fetch artifacts"
+      - func: "run-perf-test"
+        vars:
+          perf-test-name: chunk-cache-reads.wtperf
+          maxruns: 1
+          wtarg: -ops ['"read"']
+      - func: "upload stats to atlas"
+        vars:
+          test-name: chunk-cache-reads.wtperf
+      - func: "upload stats to evergreen"
+        vars:
+          test-name: chunk-cache-reads.wtperf
+
+  - name: perf-test-chunk-cache-base
+    tags: ["chunk-cache-perf"]
+    depends_on:
+      - name: compile
+    commands:
+      - func: "fetch artifacts"
+      - func: "run-perf-test"
+        vars:
+          perf-test-name: chunk-cache-reads-base.wtperf
+          maxruns: 1
```

Comment:
shall we run this more than once? Perhaps 3? 

That should give us a better average.

Reply:
Let's wait and see how noisy the chart turns out to be? If we can get away with not burning 3x the compute resources, that'd be good.